static/js/inventory.js

```diff
@@ -114,6 +114,51 @@ function submit_category() {
     });
 }
 
+function stock_update(SN) {
+    let stock = document.getElementById("stock_amount").value;
+
+    fetch("/product/change_stock", {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify({
+            "SN": SN,
+            "stock": stock
+        })
+    }).then(response => {
+        if (response.status == 200) {
+            window.location.href = "/";

```

Comment:
I think that it is more logical to just refresh the current page after the alert since an admin might want to do more than 1 things to a product

Reply:
Yes, that might be better for an admin. Fixed the problem.